pages/entity/_type/_.vue

```diff
@@ -172,9 +173,9 @@
         };
       },
       title() {
-        if (!this.entity) return this.$t('entity');
-        if (this.editorialTitle) return this.editorialTitle;
-        return this.entity.prefLabel[this.$store.state.i18n.locale];
+        if (!this.entity) return this.titleFallback(this.$t('entity'));
```

Comment:
This will always be in the user's UI language, so does not need the lang attribute.